Dockerfile

```diff
@@ -0,0 +1,86 @@
+FROM node:14-alpine as node-build
+WORKDIR /code
+
+COPY package.json package-lock.json ./
+RUN npm install
+
+RUN mkdir public && mkdir public/scss && mkdir public/css
+
+COPY public/scss/ ./public/scss/
+RUN npm run scss
+
+FROM php:5.6-fpm as php-target
+
+RUN apt-get update && apt-get upgrade -y
+
+RUN apt-get install -y \
+        cron \
+        git \
+        libfreetype6-dev \
+        libicu-dev \
+        libjpeg62-turbo-dev \
+        libmagickwand-dev \
+        libmemcached-dev \
+        libpng-dev \
+        libpq-dev \
+        libxml2-dev \
+        unzip \
+        zlib1g-dev
+
+RUN docker-php-ext-configure gd --with-freetype-dir=/usr/include/ --with-jpeg-dir=/usr/include/ \
+    && docker-php-ext-install \
```

Comment:
You should add something like opcache, will make things a lot faster. Also since the code doesn't change you can disable the opcache checks for changed files I think.

Reply:
I included memcached

Reply:
memcached and opcache are two completely different things..
memcached caches data, opcache caches php opcode so php doesn't have to parse your php files on every request

Reply:
I did some googling, and it turns out they indeed are not the same! Also, thanks for the recommendations @rinkp . I would however suggest to increase `opcache.revalidate_freq` to for example `60` seconds. Zero implies that cache is always revalidated and this is probably not what you want in production. Next commit will include opcache.

Reply:
In production, we have `opcache.validate_timestamps = 0` and then `opcache.revalidate_freq` is ignored. In development we don't really care about performance, so there each request is fine.

Reply:
I verified that the server response when requesting the frontpage is now just as fast as for the current production website. It used to be three times as high XD

Reply:
> In production, we have `opcache.validate_timestamps = 0` and then `opcache.revalidate_freq` is ignored. 

Indeed, in production your code should never change, so there’s no reason to every invalidate the cache.